### ISSUE TITLE
Parser error in compress.liquid

### ISSUE THREAD

**Issue by dancingmonkey** [penibelst/jekyll-compress-html#105]
I'm using Jekyll 4.3.1

When parsing the file `compress.liquid` it gives these two errors: 
`Parse error (line 106) on: <pre{{ _pres.first }}<`
`Parse error (line 107) on: <pre{{ _pres.first }}<`

Not a very deep understand of liquid syntax so I'm not creating a pull request.

**Comment by jnvsor** [comment]
Yeah that's the uncompiled file, you want to use https://github.com/penibelst/jekyll-compress-html/releases/download/v3.1.0/compress.html

That said, there have been unconfirmed reports (#104) that some of this breaks in jekyll 4 but I can't reproduce it myself.

Can you try it with that one and see if it works for you?
